2-positioning/styles.css

```
@@ -0,0 +1,48 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+header {
+  position: sticky;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  top: 0px;
+  border: 2px solid rgb(0, 255, 85);
+  color: aliceblue;
+  width: 100vw;
+  height: 60px;
+  text-align: center;
+  background-color: black;
+}
+
+.wrapper {
+  display: flex;
+}
+
+nav {
+  position: absolute;
+  width: 200px;
+  border: 2px solid blue;
+  z-index: -1;
```

```suggestion
  z-index: 1;
```
Avoid negative and super large numbers when using z-index, these values should be under your control, if you need to make sure it is above/below another element, just add one more unit to the default z-index